src/VisualStudio/Core/Def/Implementation/PickMembers/PickMembersDialogViewModel.cs

```diff
@@ -20,7 +20,7 @@ internal class PickMembersDialogViewModel : AbstractNotifyPropertyChanged
             IGlyphService glyphService,
             ImmutableArray<ISymbol> members)
         {
-            MemberContainers = members.Select(m => new MemberSymbolViewModel(m, glyphService)).OrderBy(s => s.MemberName).ToList();
+            MemberContainers = members.Select(m => new MemberSymbolViewModel(m, glyphService)).ToList();
```

Comment:
Oh, I agree. I was just pointing out how you were sneaking in design changes. Gotta keep my eye on you. :smile: